portfolio/src/main/webapp/script.js

```diff
@@ -50,8 +50,64 @@ function showPicture(){
   imageContainer.appendChild(imgElement);    
 }
 
-function sayHello(){
-  fetch('/data').then(response => response.text()).then((statement) => {
-    document.getElementById('server-container').innerText = statement;
+/**
+ * Alerts the user when they have failed to fill out all required
+ * input fields.
+ */
+function required() {
```

Comment:
When I try to submit a comment with empty name or content, it just refreshes the page without showing the alert. I think this worked yesterday though, so maybe it's a problem with the next PR?